docs/source-map-how-to.asciidoc

```diff
@@ -124,10 +124,10 @@ curl -X POST "http://localhost:5601/api/apm/sourcemaps" \
 -H 'Content-Type: multipart/form-data' \
 -H 'kbn-xsrf: true' \
 -H 'Authorization: ApiKey ${YOUR_API_KEY}' \ <2>
--F 'service_name="foo"' \
--F 'service_version="$SERVICEVERSION"' \
--F 'bundle_filepath="/test/e2e/general-usecase/bundle.js.map"' \
--F 'sourcemap="@./dist/app.min.js.map"'
+-F 'service_name=foo' \
+-F 'service_version=$SERVICEVERSION' \
+-F 'bundle_filepath=/test/e2e/general-usecase/bundle.js.map' \
```

Comment:
I dont think the bundle_path should be pointing the .map but the actual .js file.

The docs is confusing because it shows .js in one example and .map in one example.

While we are at this I think someone need to clarify this

Reply:
you are right, it should point to the .js file 👍

Reply:
@LHB6540 while you're at it, would you mind also removing the `map` part from `bundle_filepath`?

Reply:
> @LHB6540 while you're at it, would you mind also removing the `map` part from `bundle_filepath`?
When I actually use it, bundle_filepath actually points to the js file instead of the js.map file. I agree with the above, it should be a bug in the documentation. Should I revoke this pr and submit it after the revision or open another pr for this error?

Reply:
It is a pure doc issue, so changing to `-F 'bundle_filepath=/test/e2e/general-usecase/bundle.js' \` with this PR would be ok.

Reply:
> It is a pure doc issue, so changing to `-F 'bundle_filepath=/test/e2e/general-usecase/bundle.js' \` with this PR would be ok.
done, commit: https://github.com/elastic/apm-server/pull/9359/commits/e0277492ac9c1bf7e30e03f10cd7c795c1eff74a